File: unittest/scripts/py_dev_api_examples/results/Working_with_SQL_Result_Sets_1.py
res = mySession.sql('CALL my_proc()').execute()

if res.has_data():
    
    row = res.fetch_one()
    if row:
        print 'List of row available for fetching.'
        while row:
            print row
            row = res.fetch_one()
    else:
        print 'Empty list of rows.'
else:
    print 'No row result.'

File: unittest/scripts/js_dev_api_examples/results/Working_with_SQL_Result_Sets_1.js

var res = mySession.sql('CALL my_proc()').execute();

if (res.hasData()){

  var row = res.fetchOne();
  if (row){
    print('List of row available for fetching.');
    do {
      print(row);
    } while (row = res.fetchOne());
  }
  else{
    print('Empty list of rows.');
  }
}
else {
  print('No row result.');
}
